ziti-springboot-client/build.gradle

```diff
@@ -33,7 +33,7 @@ dependencies {
     implementation("org.springframework.boot:spring-boot:${springbootVersion}")
     implementation("org.springframework.boot:spring-boot-autoconfigure:${springbootVersion}")
     implementation("org.springframework.boot:spring-boot-starter-web:${springbootVersion}")
-    implementation("org.apache.httpcomponents.client5:httpclient5:5.3.1")
+    implementation('org.apache.httpcomponents.client5:httpclient5:5.3')
```

Comment:
nit: it'd be nice to keep the quotes as quotes. it sticks  out that this is `'` not `"`